test/fixtures/RetrieveUkPropertyPeriodSummary/ResponseModelsFixture.scala

```diff
@@ -210,6 +210,7 @@ trait ResponseModelsFixture {
     submittedOn = "2020-06-17T10:53:38Z",
     fromDate = "2019-01-29",
     toDate = "2020-03-29",
+    None,
```

Comment:
should also include the creation date here and the fullDownstreamJson for completeness. This would be reflected in the fullMtdJson